OpenRA.Mods.Common/Traits/Multipliers/DamageMultiplier.cs

```diff
@@ -0,0 +1,34 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using OpenRA.GameRules;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Damage taken by this actor is multiplied based on upgrade level.",
+		"Decrease to increase actor's apparent strength.",
+		"Use 0 to make actor invulnerable.")]
+	public class DamageMultiplierInfo : UpgradeMultiplierTraitInfo, ITraitInfo
+	{
+		public DamageMultiplierInfo()
+			: base(new int[] { 91, 87, 83, 65 }) { }
```

Comment:
0 causes different behaviour from being disabled which could help debug mods if mistakes are made by making the error (missing `Modifier` field) apparent during game-play, while 100 would require noticing that the multiplier is present and its `Modifier` field is missing.

In short, 0 makes a mistake more apparent to anyone who can play the mod with a distinct error.